CLDConfig/CLDReconstruction.py

```diff
@@ -307,6 +307,13 @@
                                  "UseTruthInPrefit": ["false"]
                                  }
 
+if any(small_vtx in detPath for detPath in geoservice.detectors for small_vtx in ["_o2_", "_o3_", "_o4_"]):
```

Comment:
As long as there is just the main XML file passed this works, of someone passes the subdetectors individually this might start to use the wrong value.

Reply:
maybe we should then simplify this to only handle one `detPath` and warn otherwise?

Reply:
Do you mean fail with an error if there is more than one detector XML file?

Reply:
yes